examples/hotrod/pkg/tracing/mux.go

```diff
@@ -40,37 +40,28 @@ func NewServeMux(copyBaggage bool, tracer opentracing.Tracer, logger log.Factory
 // TracedServeMux is a wrapper around http.ServeMux that instruments handlers for tracing.
 type TracedServeMux struct {
 	mux         *http.ServeMux
-	copyBaggage bool
-	tracer      opentracing.Tracer
+	tracer      trace.TracerProvider
 	logger      log.Factory
+	req         http.Request
```

Comment:
`TracedServeMux` is a type of HTTP Handler which deals with many requests, it cannot have a member variable like this.